net/config.go

```diff
@@ -64,3 +65,10 @@ func WithListenAddresses(addresses ...string) NodeOpt {
 		opt.ListenAddresses = addresses
 	}
 }
+
+// WithBootstrapPeers sets the bootstrap peer addresses to attempt to connect to.
+func WithBootstrapPeers(peers ...string) NodeOpt {
+	return func(opt *Options) {
+		opt.BootstrapPeers = peers
+	}
+}
```

Comment:
suggestion: All other config functions are tested in this file except this one. Please add a test if not too much hassle.

Reply:
added a test here